src/cpu/x64/brgemm/jit_brgemm_amx_uker.cpp

```diff
@@ -186,6 +186,8 @@ struct jit_brgemm_amx_uker_base_t : public jit_base_brgemm_kernel_t {
     const reg64_t reg_converted_stride = rsi;
     const reg64_t reg_zp_comp_pad_a = rsi;
 
+    const reg64_t reg_long_offt = r11;
```

Comment:
```suggestion
    const reg64_t reg_long_offt = reg_A;
```

And a comment why it's safe to use `reg_A` at the point it's used would be nice.

Reply:
Honestly, I think creating an alias using another alias is a bad practice. Leaving a comment on why it's ok to share r11 should be sufficient and not confusing.

Reply:
Just followed the consistent practice, but I don't mind. Wish we had an allocator for that...

Reply:
@dzarukin i assume that this is Caller-saved register according to System V ABI.

Reply:
If you were replying to allocator piece, that's not what I meant. An allocator supposed to solve a manual register search problem and find a free one at runtime without developer's attention.